### PR TITLE
meta: speed up and reduce memory for loading metadata

### PR DESCRIPTION
**Pull request by SandyXSD**

### CODE REVIEW

File: pkg/meta/sql.go [hunk 1]
```diff
@@ -3133,155 +3092,95 @@ func (m *dbMeta) LoadMeta(r io.Reader) error {
...
+	chs[5] <- &counter{totalInodes, counters.UsedInodes}
+	chs[5] <- &counter{"nextInode", counters.NextInode}
+	chs[5] <- &counter{"nextChunk", counters.NextChunk}
+	chs[5] <- &counter{"nextSession", counters.NextSession}
+	chs[5] <- &counter{"nextTrash", counters.NextTrash}
+	for _, d := range dm.DelFiles {
+		chs[5] <- &delfile{d.Inode, d.Length, d.Expire}
 	}
+	for k, v := range refs {
+		chs[3] <- &chunkRef{k.id, k.size, int(v)}
```
Comment (davies) [inline pkg/meta/sql.go:3160]: we should insert those with ref =1, then update them if ref > 1

Comment (SandyXSD) [reply pkg/meta/sql.go:3160]: `chunkRef` is not inserted until in this for loop, so no need to update

Comment (SandyXSD) [reply pkg/meta/sql.go:3160]: This could be an issue, if there are too many keys in `refs`.

Comment (davies) [reply pkg/meta/sql.go:3160]: there could be more rows than node

**Pull request merged**